src/ziggurat/config.clj

```diff
@@ -72,3 +72,9 @@
 
 (defn get-in-config [ks]
   (get-in (ziggurat-config) ks))
+
+(defn channel-retry-config [topic-entity channel]
+  (get-in (ziggurat-config) [:stream-router topic-entity :channels channel :retry]))
+
+(defn retry-config []
+  (get-in (ziggurat-config) [:retry]))
```

Comment:
`get-in` is not required here. you can use `get` instead

Reply:
the function is removed